site/src/pages/release-notes/1.14.0.mdx

```diff
@@ -0,0 +1,337 @@
+            .get("/v1/items/1")
+            .asJson(MyObject.class)
+            .execute();
+  ``` 
+  <Tip>
+
+  Note that you should never use this client in an `EventLoop` thread.
+  Use it from a non-`EventLoop` thread such as <type://BlockingTaskExecutor>.
+ 
+  </Tip>
+- You can now fluently build a gRPC client with various options using
+  <type://GrpcClientBuilder>. #3981 #3999 #3964 #3975
+  ```java
+  GrpcClients.builder(...)
+             .decorator(myDecorators)
+             .maxResponseMessageLength(MAX_MESSAGE_SIZE)
+             .jsonMarshallerFactory(descriptor -> {
+                 ...
+             })
+             .intercept(myInterceptors)
+             .build(MyStub.class);
+  ``` 
+- You can now fluently build an <type://HttpResponse> by using <type://HttpRequestBuilder>. #3398 #3941
+  ```java
+  HttpResponse.builder()
+              .ok()
+              .header(HttpHeaderNames.USER_AGENT, "Armeria")
+              .content(content)
+              .build(); 
+  ```
+- You can now use <type://StreamMessage#mapAsync(Function)> to transform a <type://StreamMessage> 
+  using an async operation. #3916 #3962
+  ```java
+  StreamMessage<Integer> userIds = StreamMessage.of(...);
+  StreamMessage<UserInfo> userInfos = streamMessage.mapAsync(this::getUserInfo);
+  
+  CompletableFuture<UserInfo> getUserInfo(int userId) {
+      ...
+  }
+  ```
+- You can now use <type://HttpMessage#peekData(Consumer)>, <type://HttpMessage#peekTrailers(Consumer)> and
+  <type://StreamMessage#peekError(Consumer)> and <type://HttpResponse#peekHeaders(Consumer)> operators on
+  <type://HttpRequest> and <type://HttpResponse>. #3097 #3988 #3949
+  ```java
+  HttpRequest request = HttpRequest.of(RequestHeaders.of(HttpMethod.POST, "/items"),
+                                       HttpData.ofUtf8("data1,data2"))
+                                       HttpHeaders.of("trailer", "foo"));
+  HttpRequest peeked = 
+    request.peekData(data -> {
+      assert data.toStringUtf8().equals("data1,data2");
+    }).peekTrailers(trailers -> {
+       assert trailers.get("trailer").equals("foo");
+    });
+  
+  HttpRequest failed = HttpRequest.ofFailure(new IllegalStateException("Something went wrong."));
+  HttpRequest peeked = failed.peekError(cause -> {
+    assert cause instanceof IllegalStateException;
+  });
+  ```
+- You can now generate an error response differently in <type://ServerErrorHandler> depending 
+  on the <type://RequestHeaders>. #4037
+- You can now use `{*var}` and `:*var` path patterns to capture rest paths of a request. #3031 #3997
+  ```java
+  Server.builder()
+        .service("/api/{item}/route/{*contents}", (ctx, req) -> {
+            // If a request path is "/api/123/route/foo/bar/baz",
+            // '*contents' should be:
+            assert ctx.pathPath("contents").equals("foo/bar/baz");
+        });
+  ```
+- You can now use <type://ServiceRequestContext#queryParams(String)> to get a decoded query part. #3955 #3960
```

Comment:
nit; I don't think I've ever heard of the term `query part` (correct me if I'm wrong 😅 )

```suggestion
- You can now use <type://ServiceRequestContext#queryParams(String)> to get the decoded query parameters. #3955 #3960
```